babel.config.js

```diff
@@ -4,11 +4,21 @@ module.exports = (api) => {
     ['@babel/plugin-transform-react-jsx', {
       runtime: 'automatic',
       importSource: 'preact',
+      useBuiltIns: true
```

Comment:
This is documented as only meaning something for `runtime: 'classic'`?

https://babeljs.io/docs/en/babel-plugin-transform-react-jsx#react-automatic-runtime-1